### PR TITLE
[OSDEV-831] API. Handle Geocode errors w/ system error code. 

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-831](https://opensupplyhub.atlassian.net/browse/OSDEV-831) API. Handle Geocode errors w/ system error code. 

- Reduced timeout for DedupeHub & Geocoding processing data
- Skipped triggering match process if geocoding failed
- Added error message
- Returned 200 status code when GEOCODED_NO_RESULTS

### CODE REVIEW

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
Could you please also update the Swagger docs to include one more possible example of the response from the /facilities POST API endpoint?

File: src/django/api/constants.py [hunk 1]
```diff
@@ -186,3 +186,7 @@ class NumberOfWorkersRanges:
         'min': 10001,
         'label': 'More than 10000'
     }]
+
+
+class ErrorMessages:
+    GEOCODING_NO_RESULTS = "The address you submitted can not be geocoded."
```
Comment (vladsha-dev) [inline src/django/api/constants.py:192]: Do you think we can name it `GEOCODED_NO_RESULTS` to have a similar naming to the status `FacilityListItem.GEOCODED_NO_RESULTS`?

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: src/django/api/constants.py (hunk 1 above)
Comment (roman-stolar) [reply src/django/api/constants.py]: Done.

**Comment by roman-stolar** [comment]
> Could you please also update the Swagger docs to include one more possible example of the response from the /facilities POST API endpoint?

field "message" already exist there in some cases, but yes I'll update

**Comment by roman-stolar** [comment]
@vladsha-dev Ready for re-review.

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by killev — APPROVED** [review]
LGTM. Approved

**Pull request merged**